<launch>

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find outdoor_bot)/param/costmap_common_params_outdoor.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find outdoor_bot)/param/costmap_common_params_outdoor.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find outdoor_bot)/param/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find outdoor_bot)/param/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find outdoor_bot)/param/base_local_planner_params_outdoor.yaml" command="load" />

    <!-- turtlebot also has this, don't know why
    <remap from="cmd_vel" to="cmd_vel_mux/input/navi_raw"/>
     -->

  </node>

</launch>
